src/frontend/src/pages/AdminToolsPage/AdminToolsUserManagement.tsx

```diff
@@ -108,13 +110,21 @@ const AdminToolsUserManagement: React.FC = () => {
             sx={styles.roleSelectStyle}
             disabled={!user}
           >
-            {Object.values(RoleEnum)
-              .filter((v) => v !== RoleEnum.APP_ADMIN)
-              .map((v) => (
-                <MenuItem value={v} key={v}>
-                  {v}
-                </MenuItem>
-              ))}
+            {!isAdmin(currentUser.role)
```

Comment:
why not get rid of the ! and then flip the ternary order so that its not like a double negative